scripts/deblur

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import click
+
+from skbio.parse.sequences import parse_fasta
+
+from deblur.deblurring import deblur
+
+
+@click.group()
+def deblur_cmds():
+    pass
+
+
+@deblur_cmds.command()
+@click.argument('seqs_fp', required=True,
+                type=click.Path(resolve_path=True, readable=True, exists=True,
+                                file_okay=True))
+@click.option('--read-error', '-e', required=False, type=float, default=0.05,
+              help="Read error rate")
```

Comment:
Those defaults are from @amnona 
As I said in the email thread; let's not worry about default values since until we finish the paper and we do a lot of testing we cannot decide which are the best default values.